allauth/socialaccount/models.py

```diff
@@ -299,6 +303,9 @@ def _lookup_by_socialaccount(self):
             signals.social_account_updated.send(
                 sender=SocialLogin, request=context.request, sociallogin=self
             )
+            get_adapter().send_notification_mail(
+                "socialaccount/email/social_account_updated", self.user, {}
```

Comment:
I don't think this notification serves any purpose. From the user's perspective, adding/removing an account is an important security concern, but updating the account data itself is not really relevant and basically happens each login. I propose to remove this.